x-pack/plugins/ml/public/application/jobs/new_job/pages/components/pick_fields_step/components/advanced_detector_modal/function_help.tsx

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState, memo, FC } from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import {
+  EuiButtonIcon,
+  EuiPopover,
+  EuiPopoverTitle,
+  EuiBasicTable,
+  EuiPanel,
+  EuiLink,
+  EuiText,
+} from '@elastic/eui';
+import { useMlKibana } from '../../../../../../../contexts/kibana';
+
+export const FunctionHelpPopover: FC = memo(() => {
+  const {
+    services: { docLinks },
+  } = useMlKibana();
+  const [isHelpOpen, setIsHelpOpen] = useState(false);
+
+  const onHelpClick = () => setIsHelpOpen((prevIsHelpOpen) => !prevIsHelpOpen);
+  const closeHelp = () => setIsHelpOpen(false);
+
+  const helpButton = <EuiButtonIcon onClick={onHelpClick} iconType="documentation" />;
+
+  const columnsSidebar = [
+    {
+      field: 'function',
+      name: 'Function',
+      width: '150px',
+    },
+    {
+      field: 'description',
+      name: 'Description',
+    },
+  ];
+
+  const items = [
+    {
+      id: 0,
+      function: 'count, high_count, low_count',
+      description: 'Detect anomalies when the number of events in a bucket is anomalous.',
+    },
+    {
+      id: 1,
+      function: 'non_zero_count, high_non_zero_count, low_non_zero_count',
+      description:
+        'Detect anomalies when the number of events in a bucket is anomalous, but it ignores cases where the bucket count is zero.',
```

Comment:
Not mandatory, but you can synch the verbs here:
```suggestion
        'Detect anomalies when the number of events in a bucket is anomalous, but ignore cases where the bucket count is zero.',
```